composer.json

```diff
@@ -1,18 +1,25 @@
 {
-    "name": "mangopay/php-sdk-v2",
-    "description": "SDK PHP for Mangopay api V2",
-    "minimum-stability": "stable",
-    "license": "MIT",
-    "require": {
-        "php": ">=5.4.0",
-        "ext-curl": "*",
-        "ext-openssl": "*",
-        "psr/log": "^1.0"
-    },
-    "autoload": {
-        "psr-4": { "MangoPay\\": "MangoPay/" }
-    },
-    "require-dev": {
-        "simpletest/simpletest": "^1.1"
+  "name": "mangopay/php-sdk-v2",
+  "description": "SDK PHP for Mangopay api V2",
+  "minimum-stability": "stable",
+  "license": "MIT",
+  "require": {
+    "php": ">=5.4.0",
+    "ext-curl": "*",
+    "ext-openssl": "*",
+    "psr/log": "^1.0"
+  },
+  "autoload": {
+    "psr-4": {
+      "MangoPay\\": "MangoPay/"
     }
-}
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "MangoPay\\Tests\\": "tests/"
+    }
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^4.8.36"
```

Comment:
This constraint should be relaxed to also allow newer versions of PHPUnit: `^4.8.36 || ^5.7.27 || 7.3` for example would allow installing PHPUnit 5 when running PHP 5.6 or later and PHPUnit 7.3 when running PHP 7.1 or later, instead of locking everyone to a relatively old version.

Reply:
Ok, I'll try that to see if there are no syntax changes between those versions of PHPUnit